components/User.tsx

```diff
@@ -0,0 +1,70 @@
+import { Avatar, Box, Flex, Text, WrapItem } from '@chakra-ui/react';
+import { useEffect, useState } from 'react';
+import { accountState } from '../store/auth';
+import { HomeSectionTitle } from './HomeSectionTitle';
+
+export const User = () => {
+  const [herotag, setHerotag] = useState('');
+  const [nonce, setNonce] = useState('');
+  const [shard, setShard] = useState('');
+
+  async function getHerotag() {
+    const res = await fetch(
+      `https://api.elrond.com/accounts/${accountState.address}`
```

Comment:
The API can't be hardcoded. There are env variables for that also, there are helpers for apiCalls in utils, and the dapp also uses SWR, which could be used here